proto/src/main/proto/Rss.proto

```diff
@@ -135,6 +136,13 @@ message GetShuffleResultRequest {
   int32 partitionId = 3;
 }
 
+message GetShuffleResultForMultiPartRequest {
+  string appId = 1;
+  int32 shuffleId = 2;
+  int32 startPartition = 3;
```

Comment:
Partitions may not be continuous.

Reply:
RssShuffleManager#getReader partition is continuous, GetShuffleResult partition is discontinuous, but it will not cause wrong results, right?

Reply:
p1、p3 on server1, p2、p4 on server2，will send GetShuffleResultForMultiPart(p1,p4) to server1 and server2

Reply:
What do you think? @jerqi

Reply:
It seems that in the case of multiple replica, redundant data will be requested